octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -247,6 +420,9 @@ async def candle(self, feed, symbol, start, stop, interval, trades, open_price,
                                            time_frame=time_frame,
                                            symbol=symbol,
                                            candle=candle)
+            await self.push_to_channel(trading_constants.TICKER_CHANNEL,
```

Comment:
I think it's a very good idea to create the ticker from here but we should only push it on one timeframe otherwise i'm affraid we will push it for each watched timeframe for each price change (because each candle of the same pair will receive an update)